data/src/main/java/com/linkedin/data/codec/entitystream/AbstractJacksonDataEncoder.java

```diff
@@ -46,12 +46,16 @@ protected AbstractJacksonDataEncoder(JsonFactory jsonFactory, DataMap dataMap, i
   {
     super(dataMap, bufferSize);
     _jsonFactory = jsonFactory;
+    // Disable field name interning since it causes GC issues.
+    _jsonFactory.disable(JsonFactory.Feature.INTERN_FIELD_NAMES);
   }
 
   protected AbstractJacksonDataEncoder(JsonFactory jsonFactory, DataList dataList, int bufferSize)
   {
     super(dataList, bufferSize);
     _jsonFactory = jsonFactory;
+    // Disable field name interning since it causes GC issues.
+    _jsonFactory.disable(JsonFactory.Feature.INTERN_FIELD_NAMES);
```

Comment:
this is shouldn't be there in abstract class right? what if someone wants to override it?
same for other places

Reply:
Safe to do it here since its the topmost. Subclasses can change this if they want to.

Reply:
This wouldn't allow someone to override the interning feature. Subclasses cannot override this behavior. 
Same for the decoder.